src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const STATUS = 'STATUS';
```

Comment:
- [ ] Kindly refactor the value of your Action Type to use the ducks pattern: `'bookStore/categories/STATUS'`

Reply:
thanks, done.